### ISSUE TITLE
Update Accessibility Documentation

### ISSUE THREAD

**Issue by AgneLukoseviciute** [facebook/react-native-website#2679]
RN[ #31532 ](https://github.com/facebook/react-native/pull/31532) added accessibility actions to Button, TextInput, Text, and Picker. Documentation should be updated to include `AccessibilityActions` and `onAccessibilityAction` props  for those components. 

Namely, [Text](https://github.com/facebook/react-native/pull/31532) and [Button](https://reactnative.dev/docs/next/button) have no mention of these props.

**Comment by Simek** [comment]
Hi @AgneLukoseviciute, thank you for creating an issue. However it looks like the commit is only a part of `master` branch and won't be avaible in `0.65` release (unless the commit will be cherry-picked). To avoid confusing people those props should be added to `next` branch after the `0.65` documentation will be cut.

CC @lunaleaps @kacieb

**Comment by lunaleaps** [comment]
@Simek I think this would be a great first issue! Will add the tag

**Comment by EvanErickson** [comment]
Can someone elaborate on what exactly is needed? I'm happy to do a PR for this but I've never used it before.

**Comment by lunaleaps** [comment]
@EvanErickson If you check out the RN docs right now we have documentation on accessibility actions: https://reactnative.dev/docs/next/accessibility#accessibility-actions
and specifically we have documentation on the relevant props on the `View` component: https://reactnative.dev/docs/next/view#accessibilityactions

What @AgneLukoseviciute mentioned is that we recently added support for the relevant accessibilityAction props to a few more components like Button, TextInput, and Text so we'd want to update the docs for those components to include the relevant props (`onAccessibilityAction`, `accessibilityActions`.. I believe it's just those two) for those components.
